### PR TITLE
Streamline release issue template

### PR DESCRIPTION
**Pull request by garethbowen**
# Description

Updated based on our discussion in the product team leaders call. If/when this passes I'll copy the templates to the other repos.

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Review by craig-landry — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/z_release_major.md [hunk 1]
```diff
@@ -7,25 +7,22 @@ assignees: ''
 
 ---
 
-# Planning - Product Manager
+# Planning - Manager
```
[the remark this answers is not in the document]
Comment (garethbowen) [reply .github/ISSUE_TEMPLATE/z_release_major.md:10]: That's a great question. I think I was trying to make it clear it wasn't the role "Product Manager" but could be anyone. Looking over the list of action items again I wonder if it's unnecessary to include a "role" as I'd be quite happy with an engineer doing it. How do you feel about removing all role info?

[the remark this answers is not in the document]
Comment (craig-landry) [reply .github/ISSUE_TEMPLATE/z_release_major.md:10]: I like having the role on there actually. I agree that anyone could do many of the tasks, but that kind of just makes it nobody's job to do. I prefer the clarity.

An option that could work both ways here would be that those roles are defaults, but people are free to override those via informal chats as needed/desired.

[the remark this answers is not in the document]
Comment (garethbowen) [reply .github/ISSUE_TEMPLATE/z_release_major.md:10]: Ok. Do you think "Product Manager" is the right role and I should revert this change?

[the remark this answers is not in the document]
Comment (craig-landry) [reply .github/ISSUE_TEMPLATE/z_release_major.md:10]: Yeah, I think that's the most logical default.

**Review by craig-landry — APPROVED** [review]

**Pull request merged**